components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtUtil.java

```diff
@@ -18,6 +18,7 @@
 
 package org.wso2.carbon.identity.application.mgt;
 
+import edu.emory.mathcs.backport.java.util.Collections;
```

Comment:
This is not the correct package for "Collections" class. should be "java.util.Collections"